WebKitBrowser/Apps.conf.in

```diff
@@ -21,7 +21,8 @@ configuration.add("touch", "false")
 configuration.add("msebuffers", "@PLUGIN_APPS_MSEBUFFERS@")
 configuration.add("thunderdecryptorpreference", "@PLUGIN_WEBKITBROWSER_THUNDER_DECRYPTOR_PREFERENCE@")
 configuration.add("memoryprofile", "@PLUGIN_WEBKITBROWSER_MEMORYPROFILE@")
-configuration.add("memorypressure", "@PLUGIN_WEBKITBROWSER_MEMORYPRESSURE@")
+configuration.add("webprocesslimit", "@PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_WEBPROCESSLIMIT@")
```

Comment:
The idea was to create all memory related config under a separate section like

"memory" :  {
    "memoryprofile" : "1234",
    "webprocesslimit" : "2222",
    "networkprocesslimit": "3333"
}

Reply:
Done.